src/foam/nanos/auth/RetrievePassword.js

```diff
@@ -82,6 +95,9 @@ foam.CLASS({
             type: this.LogLevel.ERROR,
             transient: true
           }));
+          if ( foam.nanos.auth.DuplicateEmailException.isInstance(err.data.exception) ) {
```

Comment:
@Nauna @microArtur Looks like this is the exception wording that needs to change. 
So in this case something to the affect that 'this account requires both email and username for password reset'  - but this may not fit. 
'both email and username required'  for something shorter.

Reply:
}).catch((err) => {
         if ( this.UserNotFoundException.isInstance(err.data.exception) ) {
              this.ctrl.add(this.NotificationMessage.create({
                err: err.data,
                type: this.LogLevel.ERROR,
                transient: true
              }));
              return;
          }
          var msg = this.ERROR_MSG;
          if ( this.DuplicateEmailException.isInstance(err.data.exception) ) {
            this.userNameVisible = true;
            msg = this.DUPLICATE_ERROR_MSG;
          }
          
          this.ctrl.add(this.NotificationMessage.create({
            message: msg,
            type: this.LogLevel.ERROR,
            transient: true
          }));
        });
**task one** - update function ^
**task two** - add 2 msgs:
   DUPLICATE_ERROR_MSG = "This account requires both email and username"
   ERROR_MSG = "Issue resetting your password. Please try again"
**task three** - make model requires: UserNotFoundException and DuplicateEmailException. So that you use this.DuplicateEmailException ... and not the full path.

Reply:
}).catch((err) => {
         if ( this.UserNotFoundException.isInstance(err.data.exception) ) {
              this.ctrl.add(this.NotificationMessage.create({
                err: err.data,
                type: this.LogLevel.ERROR,
                transient: true
              }));
              return;
          }
          var msg = this.ERROR_MSG;
          if ( this.DuplicateEmailException.isInstance(err.data.exception) ) {
            this.userNameVisible = true;
            msg = this.DUPLICATE_ERROR_MSG;
          }
          
          this.ctrl.add(this.NotificationMessage.create({
            message: msg,
            type: this.LogLevel.ERROR,
            transient: true
          }));
        });
**task one** - update function ^
**task two** - add 2 msgs:
   DUPLICATE_ERROR_MSG = "This account requires both email and username"
   ERROR_MSG = "Issue resetting your password. Please try again"
**task three** - make model requires: UserNotFoundException and DuplicateEmailException. So that you use this.DuplicateEmailException ... and not the full path.